cypress/pageObjects/personContactPage.js

```diff
@@ -46,20 +46,8 @@ class PersonContactPageObjects {
         return cy.get('#contact-details-phone-number-field')
     }
 
-    phoneNumberMobileType() {
-        return cy.get('#contact-details-phone-type-mobile')
-    }
-
-    phoneNumberHomeType() {
-        return cy.get('#contact-details-phone-type-home')
-    }
-
-    phoneNumberWorkType() {
-        return cy.get('#contact-details-phone-type-work')
-    }
-
-    phoneNumberOtherType() {
-        return cy.get('#contact-details-phone-type-other')
```

Comment:
These options are not referenced at all (only `phoneNumberMobileType` was used), so I've removed them all.